<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deposit</class>
 <widget class="QWidget" name="deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>158</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="take">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string> withdrawals</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="day_2">
      <property name="text">
       <string>1 day</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="month_2">
      <property name="text">
       <string>1 month</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="half_2">
      <property name="text">
       <string>half of year</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="year_2">
      <property name="text">
       <string>1 year</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="day_4">
      <property name="text">
       <string>off</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string>how much </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>120</y>
     <width>174</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="put">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>replenishment</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="day_3">
      <property name="text">
       <string>1 day</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="month_3">
      <property name="text">
       <string>1 month</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="half_3">
      <property name="text">
       <string>half of year</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="year_3">
      <property name="text">
       <string>1 year</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="day_5">
      <property name="text">
       <string>off</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="text">
       <string>how much </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>(13% per month)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="total">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>total</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>168</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="interest_cap">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>interest</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="day">
      <property name="text">
       <string>1 day</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="month">
      <property name="text">
       <string>1 month</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="half">
      <property name="text">
       <string>half of year</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="year">
      <property name="text">
       <string>1 year</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="interest">
      <property name="text">
       <string>off</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>144</width>
     <height>120</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="input">
    <item>
     <widget class="QLineEdit" name="sum">
      <property name="text">
       <string>sum</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="time">
      <property name="text">
       <string>time</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="percent_plus">
      <property name="text">
       <string>percent_plus</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nolog_percent">
      <property name="text">
       <string>nolog_percent </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>211</width>
     <height>89</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="output">
    <item>
     <widget class="QLineEdit" name="plus_percent">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="total_sum">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="total_nolog">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>plus_percent</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>total_sum</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>total_nolog</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
